pkg/operator/ceph/cluster/cluster.go

```diff
@@ -224,20 +258,19 @@ func (c *ClusterController) initializeCluster(cluster *cluster, clusterObj *ceph
 
 	// Start the monitoring if not already started
 	c.startClusterMonitoring(cluster, cephUser)
+
+	// Start the bucket and client controller if not already started
+	c.startClusterWatchers(cluster, cephUser)
```

Comment:
Why not include all of these in the `startClusterMonitoring()` method? Otherwise they won't be triggered until after the first reconcile is completed after an operator restart. See on line 247 where `startClusterMonitoring()` is called before the first reconcile if the cluster is already configured.

Reply:
?

Reply:
Oops yes yes!